### ISSUE TITLE
[Concept] : Chassis Machine Learning - k8s Edge Node

### ISSUE THREAD

**Issue by h0lybyte** [KBVE/kbve.com#57]
**Core Concept/Theory**
- Python ML Container Service on k8s1058
- `Turn your machine learning models into portable container images that can run just about anywhere using Chassis. For a deeper dive and better understanding of what Chassis is, learn more [here](https://chassis.ml/conceptual-guides/overview/).`

* * *

**Alternative Ideas**
We could use pD to run our python scripts as a microservice. However 

* * *

**Alternative Examples/Sources**
- Core Repo : https://chassis.ml/

* * *

**Additional information**
- init - Helm @ 10/1/2022 2am

**Comment by h0lybyte** [comment]
Running a simple backtracing algo for S&P500 on a small instance of an edge k8s / k3s. 

Load Time: 
- It takes a while for the container to spin up, about 170 seconds on an i7 with about 10gb free ram.
- Its even worse for a VM that is on the windows PC lol

As for the total emulation, with only 15min volumne checks, there are no issues with moving averages but the RSI is definitely incorrect, by a long shot.

I am not too sure if there is too much nest emulation OR if I am setting up the instance incorrectly. Going to mess around with it a bit more sometime this weekend.

**Comment by h0lybyte** [comment]
Currently we are facing an issue tracking the deployment of chassis-mL tasks through our edge cloud framework. Deploying 1 has no issues but once we go 1+N, we are having a bit trouble tracking the execution. So singular instances have no issue but running multiple in parallel has some issues, but they look like they can be resolved in a couple different ways. 

The first method would be to just track the kNs instance from the start to the end. I.e create a fresh spin, apply the data / matrix , execute , push data and then kill. The second method would be to have a revolving instance of kNs that are always running and having it resolve tasks. While the 2nd method would have more speed, the draw back would be that the data might get cross contaminated or leaked. I would have to look through it all again and remap it.

**Comment by h0lybyte** [comment]
I am going to close this as its done for the most part. 

We can open a new ticket this during the offering of the service via SaaS.

**Comment by h0lybyte** [comment]
Automating Tasker
* * *
[KBVE Support](https://kbve.com/support/)
* * *
[KBVE Discord](https://kbve.com/discord/)
* * *
[KBVE Staff](https://kbve.com/team/)
* * *
[KBVE VIP Priority](https://kbve.com/vip/) || [KBVE Community](https://kbve.com/c/)
* * *
[HubSpot Ticket Created](https://app.hubspot.com/contacts/20854121/ticket/1546826607)
* * *
